Spoiler.py

```diff
@@ -47,6 +47,7 @@ class Spoiler(object):
 
     def __init__(self, worlds):
         self.worlds = worlds
+        worlds[0].settings.settings_string = worlds[0].settings.get_settings_string() # Update settings string in case settings changed during generation.
```

Comment:
Is this safe to apply with randomize_settings? Normally we don't want to update the settings string after it sets the various random settings.

Reply:
Good point I didn't think about. Turns out not safe, it will update the string based on the resolved settings. Maybe this should just be removed? On second thought it might make more sense to leave the settings string as whatever was put into the generator anyway.

Reply:
The settings string is for the initial settings, not the resolved settings, so yeah, this line should be removed.